test/controllers/APIControllerSpec.scala

```diff
@@ -51,87 +49,11 @@ class APIControllerSpec extends SpecBase with FeatureSwitching {
-        contentAsString(result) shouldBe "VRN: 1234567891234567890 was not in a valid format."
-      }
-
-      s"return OK (${Status.OK}) when the call returns some data and can be parsed to the correct response" in new Setup {
-        when(mockETMPService.checkIfHasAnyPenaltyData(any())).thenReturn(true)
-        when(mockETMPService.getNumberOfEstimatedPenalties(any())).thenReturn(2)
-        when(mockETMPService.getPenaltyDataFromETMPForEnrolment(any())(any()))
-          .thenReturn(Future.successful((Some(mockETMPPayloadForAPIResponseData), Right(GetETMPPayloadSuccessResponse(mockETMPPayloadForAPIResponseData)))))
-        when(mockETMPService.findEstimatedPenaltiesAmount(any()))
-          .thenReturn(BigDecimal(123.45))
-        when(mockETMPService.getNumberOfCrystalizedPenalties(any())).thenReturn(2)
-        when(mockETMPService.getCrystalisedPenaltyTotal(any())).thenReturn(BigDecimal(288))
-        val result = controller.getSummaryDataForVRN("123456789")(fakeRequest)
-        status(result) shouldBe Status.OK
-        contentAsJson(result) shouldBe Json.parse(
-          """
-            |{
-            |  "noOfPoints": 2,
-            |  "noOfEstimatedPenalties": 2,
-            |  "noOfCrystalisedPenalties": 2,
-            |  "estimatedPenaltyAmount": 123.45,
-            |  "crystalisedPenaltyAmountDue": 288,
-            |  "hasAnyPenaltyData": true
-            |}
-            |""".stripMargin
-        )
-        verify(mockAuditService, times(1)).audit(any())(any(), any(), any())
-      }
-
-      s"return OK (${Status.OK}) when there are no LSP or LPP estimated penalties in etmpPayload" in new Setup {
-        when(mockETMPService.checkIfHasAnyPenaltyData(any())).thenReturn(false)
-        when(mockETMPService.getNumberOfEstimatedPenalties(any())).thenReturn(0)
-        when(mockETMPService.getPenaltyDataFromETMPForEnrolment(any())(any()))
-          .thenReturn(Future.successful((Some(mockETMPPayloadWithNoEstimatedPenaltiesForAPIResponseData), Right(GetETMPPayloadSuccessResponse(mockETMPPayloadWithNoEstimatedPenaltiesForAPIResponseData)))))
-        when(mockETMPService.findEstimatedPenaltiesAmount(any()))
-          .thenReturn(BigDecimal(0))
-        when(mockETMPService.getNumberOfCrystalizedPenalties(any())).thenReturn(0)
-        when(mockETMPService.getCrystalisedPenaltyTotal(any())).thenReturn(BigDecimal(0))
-        val result = controller.getSummaryDataForVRN("123456789")(fakeRequest)
-        status(result) shouldBe Status.OK
-        contentAsJson(result) shouldBe Json.parse(
-          """
-            |{
-            |  "noOfPoints": 4,
-            |  "noOfEstimatedPenalties": 0,
-            |  "noOfCrystalisedPenalties": 0,
-            |  "estimatedPenaltyAmount": 0,
-            |  "crystalisedPenaltyAmountDue": 0,
-            |  "hasAnyPenaltyData": false
-            |}
-            |""".stripMargin
-        )
-      }
-    }
 
     "call API 1812 when call 1812 feature is enabled" must {
```

Comment:
Could you remove "call API 1812 when call 1812 feature is enabled" must { part only